### PR TITLE
[s] Prevents players from doing terrible things.

### PR DESCRIPTION
**Pull request by ghost**
See Title. See Changes.

### CODE REVIEW

**Comment by FantasticFwoosh** [comment]
**J U S T I C E**

File: code/game/gamemodes/changeling/powers/fakedeath.dm [hunk 1]
```diff
@@ -18,7 +18,7 @@
 	user.update_stat()
 	user.update_canmove()
 
-	addtimer(src, "ready_to_regenerate", LING_FAKEDEATH_TIME, TIMER_NORMAL, user)
+	addtimer(src, "ready_to_regenerate", LING_FAKEDEATH_TIME, TIMER_UNIQUE, user)
```
Comment (Chnkr) [inline code/game/gamemodes/changeling/powers/fakedeath.dm]: Players where previously able to keep the `alert()` open to call the `addtimer` again, which previously caused it to simply add two timers, allowing changelings to revive themselves whilst alive.

**Comment by KorPhaeron** [comment]
Do we need a changelog for these things?

**Comment by Chnkr** [comment]
@KorPhaeron yeah no, maybe not. Removed them.

File: code/game/gamemodes/changeling/powers/fakedeath.dm (hunk 1 above)
Comment (optimumtact) [inline code/game/gamemodes/changeling/powers/fakedeath.dm]: Is this globally unique or per user?

Comment (Chnkr) [inline code/game/gamemodes/changeling/powers/fakedeath.dm]: This is unique per user! `addtimer` creates an internal `hashlist` which is used to determine whether or not the `timer` is unique.

`hashlist[1] = "[thingToCall](\ref[thingToCall])"`

Comment (optimumtact) [inline code/game/gamemodes/changeling/powers/fakedeath.dm]: ah yeah, that was teh thing MSO fixed recently

👍

**Pull request merged**